bash/atomic

```diff
@@ -662,7 +663,7 @@ _atomic() {
 		info
 		install
 		images
-		migrate
```

Comment:
can't we leave `migrate` as a valid command in the code so ppl using `atomic migrate` won't get hurt by its replacement to `atomic storage`? no big deal though